server/src/auth/auth.controller.ts

```diff
@@ -1,26 +1,42 @@
-import { Controller, Post, Request, UseGuards } from '@nestjs/common';
+import { Controller, Post, Response, Request, UseGuards, Get } from '@nestjs/common';
 
 import { LoginAuthGuard } from './guards/login-auth.guard';
+import { CookieAuthGuard } from './guards/cookie-auth.guard';
 import { AuthService } from './auth.service';
+import { COOKIE_KEY } from './constants';
+
+import type { User } from '../users/entities/user.entity';
 
 @Controller('auth')
 export class AuthController {
   constructor(private authService: AuthService) {}
 
-  @Post('test')
+  @Post('login')
   @UseGuards(LoginAuthGuard)
-  async authTest() {
-    return {
-      message: 'ヽ(•‿•)ノ',
-    };
+  async login(@Request() request, @Response({ passthrough: true }) response): Promise<void> {
+    const { user } = request;
+    const jwt = await this.authService.createJwt(user);
+    response
+      .cookie(COOKIE_KEY, jwt, {
+        domain: 'localhost',
+        expires: new Date(new Date().getTime() + 60 * 60 * 1000), // 1 hour
+        httpOnly: true,
+        path: '/',
+        secure: true,
+        signed: true,
+      })
+      .send({ user });
```

Comment:
we probably need to set the `samesite` attribute to `strict` to avoid having the auth cookie sent along with 3rd party requests: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite#strict.